napari/layers/shapes/shapes.py

```diff
@@ -58,6 +59,33 @@
 DEFAULT_COLOR_CYCLE = np.array([[1, 0, 1, 1], [0, 1, 0, 1]])
 
 
+_REV_SHAPE_HELP = {
+    trans._('hold <space> to pan/zoom'): {
+        Mode.VERTEX_INSERT,
+        Mode.VERTEX_REMOVE,
+        Mode.ADD_RECTANGLE,
+        Mode.ADD_ELLIPSE,
+        Mode.ADD_LINE,
+    },
+    trans._('hold <space> to pan/zoom, press <esc> to finish drawing'): {
+        Mode.ADD_PATH,
+        Mode.ADD_POLYGON,
+    },
+    trans._(
+        'hold <space> to pan/zoom, press <{BACKSPACE}> to remove selected',
+        BACKSPACE=BACKSPACE,
+    ): {Mode.SELECT, Mode.DIRECT},
+    trans._('enter a selection mode to edit shape properties'): {
+        Mode.PAN_ZOOM
+    },
+}
+
+_FWD_SHAPE_HELP = {}
```

Comment:
Can you add a comment here on why this was needed just so we don't forget. I could imagine coming back to this in a while and being confused

Reply:
Done.